src/lib/forward.ts

```diff
@@ -5,24 +5,45 @@ import {
   ZWaveNodeEvents,
   ZWaveNodeMetadataUpdatedArgs,
 } from "zwave-js";
-import { CommandClasses, ConfigurationMetadata } from "@zwave-js/core";
+import {
+  CommandClasses,
+  ConfigurationMetadata,
+  ZWaveLogInfo,
+} from "@zwave-js/core";
 import { OutgoingEvent } from "./outgoing_message";
 import { dumpConfigurationMetadata, dumpMetadata, dumpNode } from "./state";
-import { Client, ClientsController } from "./server";
+import { Client, ClientsController, ZwavejsServer } from "./server";
 
 export class EventForwarder {
   /**
    * Only load this once the driver is ready.
    *
    * @param clients
    */
-  constructor(public clients: ClientsController) {}
+  constructor(
+    public clients: ClientsController,
+    private server: ZwavejsServer
+  ) {}
 
   start() {
     this.clients.driver.controller.nodes.forEach((node) =>
       this.setupNode(node)
     );
 
+    // Forward logging events on to clients that are currently
+    // receiving logs
+    this.server.on("logging", (info: ZWaveLogInfo) => {
```

Comment:
Depending what you decide below (emit message or entire info), this could also be `message: string`

Reply:
I'm going to stick with this for now so that I can see the raw data come in locally and will adjust based on what makes sense